app/javascript/gobierto_data/webapp/components/Dataset.vue

```diff
@@ -1,111 +1,122 @@
 <template>
   <div>
-    <div class="pure-g">
-      <div class="pure-u-1-2">
-        <h2
-          v-if="titleDataset"
-          class="gobierto-data-title-dataset"
-        >
-          {{ titleDataset }}
-        </h2>
+    <template v-if="!isDatasetLoading">
```

Comment:
I think this condition is in the wrong direction: if the dataset is loading, show the spinner.  Do you mean isDatasetLoaded?

Reply:
You're right.